cmd/node/main.go

```diff
@@ -372,6 +373,13 @@ VERSION:
 		Name:  "import-db-no-sig-check",
 		Usage: "This flag, if set, will cause the signature checks on headers to be skipped. Can be used only if the import-db was previously set",
 	}
+
+	// redundancyLevel defines a flag that specifies the level of redundancy for the current instance (0 = master)
+	redundancyLevel = cli.Uint64Flag{
+		Name:  "redundancy-level",
+		Usage: "This flag specifies the level of redundancy used by the current instance for the node (0 = master, 1 = first slave, 2 = second slave, etc.)",
+		Value: 0,
+	}
```

Comment:
This PR does not solve the shuffle out mechanism for the redundancy nodes. Without this shuffle out this mechanism won't work.

Reply:
It is solved as the node will run with the same ,pem so it will do the shuffle out mechanism

Reply:
Right, I thought the backup would have be ran as pure observers. By reusing the same key on multiple machines we simplified the process and impacted the code as minimum as we could get. :+1: